packages/generator/src/action-function-import/function.ts

```diff
@@ -6,19 +6,37 @@ import { getRequestBuilderArgumentsBase } from './request-builder-arguments';
 
 const parameterName = 'parameters';
 
+function functionImportReturnType(
+  functionImport: VdmFunctionImport,
+  oDataVersion: string
+) {
+  let type = functionImport.returnType.returnType;
+
+  if (functionImport.returnType.isCollection) {
+    type = `${type}[]`;
+  }
+  if (functionImport.returnType.isNullable) {
+    type = `${type}|null`;
+  }
```

Comment:
Same:
```suggestion
  if (functionImport.returnType.isCollection) {
    type = `${type}[]`;
  }
  
  if (functionImport.returnType.isNullable) {
    type = `${type} | null`;
  }
```

Reply:
Please note that this function is duplicate except for one word. Maybe there is potential to reduce duplication?